specification/azurestackhci/resource-manager/readme.typescript.md

```diff
@@ -6,8 +6,9 @@ Please also specify `--typescript-sdks-folder=<path to root folder of your azure
 ```yaml $(typescript)
 typescript:
   azure-arm: true
-  package-name: "azurestackhci"
-  output-folder: "$(typescript-sdks-folder)/packages/azurestackhci"
+  package-name: "@azure/arm-azurestackhci"
+  output-folder: "$(typescript-sdks-folder)/sdk/azurestackhci/azurestackhci"
```

Comment:
Should this be output-folder: "$(typescript-sdks-folder)/sdk/azurestackhci/arm-azurestackhci"?